cherab/openadas/repository/create.py

```diff
@@ -147,6 +147,7 @@ def populate(download=True, repository_path=None, adas_path=None):
             (carbon,    0, 'adf15/pec96#c/pec96#c_vsu#c0.dat'),
             (carbon,    1, 'adf15/pec96#c/pec96#c_vsu#c1.dat'),
             (carbon,    2, 'adf15/pec96#c/pec96#c_vsu#c2.dat'),
+            (carbon,    5, 'adf15/pec96#c/pec96#c_pju#c5.dat'),
```

Comment:
Can you add a note in the changelog that this carbon line is added to `cherab.openadas.repository.populate`, and therefore that users will need to rerun this function before running the demo?

Reply:
Sure, it's done.